usaspending_api/database_scripts/matview_generator/universal_transaction_matview.json

```diff
@@ -103,266 +91,192 @@
-    },{
+    }, {
       "name": "pop_county_code",
       "where": "\"pop_county_code\" IS NOT NULL",
       "columns": [{"name": "\"pop_county_code\""}]
-    },{
+    }, {
       "name": "pop_zip5",
       "where": "\"pop_zip5\" IS NOT NULL",
       "columns": [{"name": "\"pop_zip5\""}]
-    },{
+    }, {
       "name": "pop_congressional_code",
       "where": "\"pop_congressional_code\" IS NOT NULL",
       "columns": [{"name": "\"pop_congressional_code\""}]
-    },{
-      "name": "latest",
-      "columns": [{"name": "\"latest_transaction_id\""}]
-    },{
+    }, {
       "name": "gin_recipient_name",
       "method": "GIN",
       "columns": [{"name": "\"recipient_name\"", "opclass": "gin_trgm_ops"}]
-    },{
+    }, {
       "name": "gin_recipient_unique_id",
       "method": "GIN",
       "columns": [{"name": "\"recipient_unique_id\"", "opclass": "gin_trgm_ops"}]
-    },{
+    }, {
       "name": "gin_parent_recipient_unique_id",
       "method": "GIN",
       "columns": [{"name": "\"parent_recipient_unique_id\"", "opclass": "gin_trgm_ops"}]
-    },{
+    }, {
       "name": "recipient_id",
       "where": "\"recipient_id\" IS NOT NULL",
       "columns": [{"name": "\"recipient_id\""}]
-    },{
+    }, {
       "name": "recipient_name",
       "where": "\"recipient_name\" IS NOT NULL",
       "columns": [{"name": "\"recipient_name\""}]
-    },{
+    }, {
       "name": "recipient_unique_id",
       "where": "\"recipient_unique_id\" IS NOT NULL",
       "columns": [{"name": "\"recipient_unique_id\""}]
-    },{
+    }, {
       "name": "parent_recipient_unique_id",
       "where": "\"parent_recipient_unique_id\" IS NOT NULL",
       "columns": [{"name": "\"parent_recipient_unique_id\""}]
-    },{
+    }, {
       "name": "awarding_toptier_agency_name",
       "where": "awarding_toptier_agency_name IS NOT NULL",
       "columns": [{"name": "\"awarding_toptier_agency_name\"", "order": "DESC NULLS LAST"}]
-    },{
+    }, {
       "name": "awarding_subtier_agency_name",
       "where": "awarding_subtier_agency_name IS NOT NULL",
       "columns": [{"name": "\"awarding_subtier_agency_name\"", "order": "DESC NULLS LAST"}]
-    },{
+    }, {
       "name": "funding_toptier_agency_name",
       "where": "funding_toptier_agency_name IS NOT NULL",
       "columns": [{"name": "\"funding_toptier_agency_name\"", "order": "DESC NULLS LAST"}]
-    },{
+    }, {
       "name": "funding_subtier_agency_name",
       "where": "funding_subtier_agency_name IS NOT NULL",
       "columns": [{"name": "\"funding_subtier_agency_name\"", "order": "DESC NULLS LAST"}]
-    },{
-      "name": "awarding_toptier_agency_abbreviation",
-      "where": "awarding_toptier_agency_abbreviation IS NOT NULL",
-      "columns": [{"name": "\"awarding_toptier_agency_abbreviation\"", "order": "DESC NULLS LAST"}]
-    },{
-      "name": "awarding_subtier_agency_abbreviation",
-      "where": "awarding_subtier_agency_abbreviation IS NOT NULL",
```

Comment:
Were these removed from a previous PR? I don't see the corresponding columns being removed in this one.

Reply:
The JSON file is a little confusing. all of these dicts are describing indexes. Since there are no queries on the agency abbreviations the indexes won't be used